.github/workflows/release.yml

```diff
@@ -5,51 +5,42 @@ on:
   push:
     branches:
       - main
-    paths:
-      - "provider/**"
-      - "sdk/**"
-      - "Makefile"
-      - ".goreleaser.yml"
   workflow_dispatch:
 
 permissions:
-  contents: write
-  id-token: write
+  contents: read
+  pull-requests: read
 
 env:
-  PROVIDER: buildkite
+  PROVIDER: proxmoxve
```

Comment:
Is this copied by accident from another provider? I think this should remain on `buildkite`.